public/app/core/components/Footer/Footer.tsx

```diff
@@ -45,7 +45,10 @@ export let getVersionLinks = (): FooterLink[] => {
     return links;
   }
 
-  links.push({ text: `v${buildInfo.version} (${buildInfo.commit})` });
+  links.push({
+    text: `v${buildInfo.version} (${buildInfo.commit})`,
+    url: `https://grafana.com/docs/grafana/latest/release-notes/release-notes-${buildInfo.version}/`
```

Comment:
At a minimum, this would need the periods replaced with hyphens for the URL to be valid.

I'm taking a look to see what we should handle pre-release versions. I'm thinking prereleases just shouldn't link to anything.

Reply:
```ts
const isPrerelease = buildInfo.version.includes('-');
links.push({
  text: `v${buildInfo.version} (${buildInfo.commit})`,
  url: isPrerelease ? `https://grafana.com/docs/grafana/latest/release-notes/release-notes-${buildInfo.version.replace(".", "-")}/` ? undefined
})
```

Reply:
Good suggestion! However, this filters beta versions as well (like [v9.0.0-beta3](https://grafana.com/docs/grafana/latest/release-notes/release-notes-9-0-0-beta3/)), doesn't it?